torch/nn/functional.py

```diff
@@ -3324,25 +3326,35 @@ def grid_sample(input, grid, mode='bilinear', padding_mode='zeros', align_corner
         The default behavior up to version 1.2.0 was ``align_corners = True``.
         Since then, the default behavior has been changed to ``align_corners = False``,
         in order to bring it in line with the default for :func:`interpolate`.
+
+    .. note::
+        ``mode='bicubic'`` is implemented by
+        [cubic covolution algorithm](https://en.wikipedia.org/wiki/Bicubic_interpolation) 
+        with `\alpha=-0.75`. It's possible to cause overshoot, in other words it can produce
```

Comment:
"It's possible to cause overshoot..."->"This algorithm may "overshoot" the range of values it's interpolating. For example, it may produce negative values or values greater than 255 when interpolating values in the range [0, 255]. Clamp the interpolated values with :func:`torch.clamp` to ensure they are within the allowed range of values."

Reply:
I am worried there are too many "value" and "range" in the description

How about this?

-> Besides, this algorithm may "overshoot" the range of values it's interpolating. For example, it may produce negative values or values greater than 255 when interpolating input in [0, 255]. Clamp the results with :func: `torch.clamp` to ensure they are within the valid range.

Reply:
That sounds better. Just take out the "besides" and start with, "This algorithm..."